src/templates/apps/short-rest.html

```diff
@@ -1,9 +1,7 @@
 <form id="short-rest-rp">
-    <p>Take a short rest? On a short rest you may spend one Resovle Point to recover all Stamina Points.</p>
-    <div class="form-group">
-        <label>Spend a Resolve Point</label>
-        <div class="form-fields">
-            <button id="spend-rp">Spend RP</button>
-        </div>
-    </div>
+    <p>{{localize 'STARFINDER.RestSDescription'}}</p>
+    <label class="checkbox">
+        <input type="checkbox" id="restoreStaminaCheckbox"> {{localize "STARFINDER.RestSActionHeader"}}
+    </label><br/>
 </form>
+<br/>
```

Comment:
Not sure that we need and explicit break here. If wee need more space at the bottom of the form it would be better to use `css` styling instead of `html` markup.

Reply:
A good point. I have adjusted the margin on the rest form so that the checkbox has a little room to breathe.